getting-started/jdbc/docker-compose-bootstrap-db.yml

```diff
@@ -22,11 +22,13 @@ services:
     # IMPORTANT: the image MUST contain the Postgres JDBC driver and EclipseLink dependencies, see README for instructions
     image: apache/polaris-admin-tool:postgres-latest
     environment:
-      polaris.persistence.type: relational-jdbc
-      quarkus.datasource.db-kind: pgsql
-      quarkus.datasource.jdbc.url: jdbc:postgresql://postgres:5432/POLARIS
-      quarkus.datasource.username: postgres
-      quarkus.datasource.password: postgres
+      - JAVA_DEBUG=true
+      - JAVA_DEBUG_PORT='*:5005'
+      - POLARIS_PERSISTENCE_TYPE=relational-jdbc
+      - QUARKUS_DATASOURCE_DB_KIND=pgsql
```

Comment:
I guess these are default value. This is probably just for my information, is it OK to ignore them? To be clear, I'm OK to keep them here.

Reply:
sure thing, there are optional as the defaults to this, the only reason to keep it here is just in case we change the defaults or there are some other overrides, this should just present E2E setup without thinking of defaults